test/Abp.Zero.SampleApp.Tests/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -577,6 +578,143 @@ public void Should_Write_History_But_Not_For_Property_If_Disabled_History_Tracki
             _entityHistoryStore.Received().Save(Arg.Is<EntityChangeSet>(s => predicate(s)));
         }
 
+        [Fact]
+        public void Should_Write_History_For_TPH_Tracked_Entities_With_One_To_Many_Relationship_Changes()
+        {
+            var studentId = CreateStudentAndGetId();
+            Resolve<IEntityHistoryConfiguration>().Selectors.Add("Selected", typeof(Student), typeof(StudentLectureNote));
+
+            WithUnitOfWork(() =>
+            {
+                var student = _studentRepository.Get(studentId);
+                var lectureNote = new StudentLectureNote()
+                {
+                    Student = student,
+                    CourseName = "Course1",
```

Comment:
since these are optional fields and we not not asserting them, can remove these assignment so that the assertion for property change is consistent with property changes count